### PR TITLE
RC66.2: Desktop equip - ignore right clicks

### PR DESCRIPTION
**Pull request by dback2**
Test Plan:
https://highfidelity.testrail.net/index.php?/cases/view/15103
Also verify you can inspect equippable entities from the marketplace.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9276/app_13036_PR_db5e5ba99beab1a45c1aa6b21794414f5eff93ce.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9276/HighFidelity-Beta-PR13036-db5e5ba99beab1a45c1aa6b21794414f5eff93ce.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9276/HighFidelity-Beta-Interface-PR13036-db5e5ba99beab1a45c1aa6b21794414f5eff93ce.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9276/HighFidelity-Beta-PR13036-db5e5ba99beab1a45c1aa6b21794414f5eff93ce.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9276/HighFidelity-Beta-Interface-PR13036-db5e5ba99beab1a45c1aa6b21794414f5eff93ce.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
Right clicks now pull up the icon to select the certificate which also displays successfully.

**Comment by conklin94122** [comment]
@dback2 Needs changes Seth requested on https://github.com/highfidelity/hifi/pull/13033

File: scripts/system/controllers/controllerModules/equipEntity.js [hunk 1]
```diff
@@ -790,7 +790,7 @@ EquipHotspotBuddy.prototype.update = function(deltaTime, timestamp, controllerDa
     };
     
     var onMousePress = function(event) {
-        if (isInEditMode()) { // don't consider any mouse clicks on the entity while in edit
+        if (isInEditMode() || event.isRightButton) { // don't consider any mouse clicks on the entity while in edit
```
Comment (sethalves) [inline scripts/system/controllers/controllerModules/equipEntity.js]: should this be  || !event.isPrimaryButton instead?

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9285/app_13036_PR_f731a2d08eed6eaede593d3fe17b23a1256b7038.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9285/HighFidelity-Beta-PR13036-f731a2d08eed6eaede593d3fe17b23a1256b7038.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9285/HighFidelity-Beta-Interface-PR13036-f731a2d08eed6eaede593d3fe17b23a1256b7038.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9285/HighFidelity-Beta-PR13036-f731a2d08eed6eaede593d3fe17b23a1256b7038.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9285/HighFidelity-Beta-Interface-PR13036-f731a2d08eed6eaede593d3fe17b23a1256b7038.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
Right click now shows the "I" icon and leads to certs of the item. Certs work as expected.

**Pull request merged**